reg_tests/test_files/nrel5MWactuatorLineAnisoGauss/nrel5MWactuatorLineAnisoGauss.yaml

```diff
@@ -131,9 +131,10 @@ realms:
         nacelle_cd: 1.0
         nacelle_area: 8.0
         air_density: 1.225
-        epsilon: [ 5.0, 2.0, 2.0 ]
+        epsilon: [ 5.0, 4.0, 4.0 ]
         epsilon_tower: 5.0
         turbine_base_pos: [ 0.0, 0.0, 0.0 ]
+        turbine_hub_pos: [ 0.0, 0.0, 90.0 ]
```

Comment:
is this addition on purpose?  was the default not good, or are we just trying to match the other case to see what happens?

Reply:
I think both your comments are going to the same thing. Yes I did need to make this change in #1320 to make things work well. I believe I had to do that so it matched the value in the openfast files? I can't remember...